client/src/components/AddCharacterInfoForm.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useEffect } from "react";
+import { withRouter } from "react-router-dom";
+import ErrorList from "./layout/ErrorList";
+import translateServerErrors from "../services/translateServerErrors";
+
+import RatingForm from "./RatingForm";
+import LookForm from "./LookForm";
+
+// take in hunterIndex, characterId, and name as props
+const AddCharacterInfoForm = (props) => {
+  const characterId = props.match.params.charId;
+  const hunterIndex = props.match.params.hunterIndex;
+
+  const [errors, setErrors] = useState([]);
+
+  const [characterData, setCharacterData] = useState({
+    name: "",
+  });
+
+  const getCharacter = async () => {
+    try {
+      const response = await fetch(`/api/v1/characters/${characterId}`);
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const body = await response.json();
+      setCharacterData({
+        ...characterData,
+        name: body.character.name,
+        hunterIndex: body.character.hunterIndex,
+      });
+      return body.character.hunterIndex;
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`);
+    }
+  };
+
+  useEffect(() => {
+    getCharacter();
+  }, []);
+
+  const getFormOptions = async (hunterIndex) => {
+    try {
+      const response = await fetch(`/api/v1/hunters/${hunterIndex}`);
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const body = await response.json();
+      setFormOptions({ ...formOptions, look: body.look, ratings: body.ratings });
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`);
+    }
+  };
+
+  useEffect(() => {
+    getFormOptions(hunterIndex);
+  }, []);
+
+  const [formOptions, setFormOptions] = useState({
+    look: null,
+    ratings: null,
+  });
+
+  const [lookData, setLookData] = useState({
+    aura: null,
+    body: null,
+    clothes: null,
+    eyes: null,
+    face: null,
+  });
+
+  const handleInputChangeLook = (event) => {
+    setLookData({ ...lookData, [event.currentTarget.name]: event.currentTarget.value });
+  };
+
+  const [ratingsData, setRatingsData] = useState({});
+
+  const handleInputChangeRating = (event) => {
+    setRatingsData(JSON.parse(event.currentTarget.value));
+  };
```

Comment:
hmmm if these were already json, I would normally suggest parsing them before you add them as the input's value. Does that make sense? Though I might be missing an element here